demisto_sdk/commands/update_release_notes/tests/update_rn_test.py

```diff
@@ -1083,7 +1084,7 @@ def test_update_docker_image_in_yml_when_RN_aleady_exists(self, mocker):
         mocker.patch.object(UpdateRN, 'get_master_version', return_value='0.0.0')
         mocker.patch.object(UpdateRN, 'identify_changed_file_type', return_value=('Test', FileType.INTEGRATION))
 
-        client = UpdateRN(pack_path="Packs/Test", update_type='minor',
+        client = UpdateRN(pack_path="Packs/Test", update_type=None,
```

Comment:
@ShahafBenYakir because this test is invalid.
Calling update-release-notes with update_type=minor when the release notes already exist shouldn't work (return a message to re-run the command without the -u flag)